src/utils.js

```diff
@@ -10,13 +10,22 @@ const __generateDate = () => {
   return dayjs().toDate()
 }
 
-const testFile= (file) => {
-
+const testFile = (file) => {
```

Comment:
Can we have a more detailed name for function - because right now it's not very clear of what it's doing

Reply:
cant think of one . it should expect a json file passed to it not to be empty

Reply:
ok, this is a more better explanation than testFile